pacai/bin/eightpuzzle.py

```diff
@@ -252,18 +252,17 @@ def createRandomEightPuzzle(moves=100):
 
 if __name__ == '__main__':
   puzzle = createRandomEightPuzzle(25)
-  print('A random puzzle:')
-  print(puzzle)
+  logging.basicConfig(format='%(levelname)s - %(asctime)s - %(message)s', level=logging.INFO)
+  logging.info('A random puzzle:\n'+ str(puzzle))
```

Comment:
Also, almost all binary operators (like `+`) should be surrounded by whitespace.